stl/inc/bit

```diff
@@ -94,20 +95,128 @@ _NODISCARD constexpr _Ty rotr(const _Ty _Val, const int _Rotation) noexcept {
     }
 }
 
-template <class _Ty, enable_if_t<_Is_standard_unsigned_integer<_Ty>, int> _Enabled>
-_NODISCARD constexpr int countl_zero(const _Ty _Val) noexcept {
+
+inline bool __Is_popcnt_supported() {

```

Comment:
We conventionally use `_Ugly` names with a single leading underscore.

Reply:
whoops